packages/x-data-grid/src/components/panel/GridPanel.test.tsx

```diff
@@ -8,6 +8,7 @@ import {
 } from '@mui/x-data-grid';
 import { GridRootPropsContext } from '@mui/x-data-grid/context/GridRootPropsContext';
 import Popper from '@mui/material/Popper';
+import describeConformance from 'test/utils/describeConformance';
```

Comment:
Nitpick: I believe that we prefer named exports over default all the time. 🤔 
WDYT?